README.md

```diff
@@ -86,8 +93,9 @@ Compile with
 ```make batchgcd```
 and **after reading the disclaimer below**, run with
 ```
-./batchgcd <path to your target csv file>
+./batchgcd <path to your target csv file> [base]
```

Comment:
Thanks for the PR, Stan!
I agree with this feature. Could you propose a more standard Unix command format, like
```
./batchgcd file.csv -base 10
```
or 
```
./batchgcd file.csv -base10
```

Reply:
Yeah sure ! Will work on that

Reply:
Done